text/0249-v2-experiments.md

```diff
@@ -0,0 +1,127 @@
+---
+feature name: V2 Experiments
+start date: 2020-09-08
+? rfc pr
+? related issue
+---
+
+# Summary
+
+When CDK v2 is released to General Availability (GA), the main release branch will no longer contain experimental code.
```

Comment:
I actually don't like this summary. How about we state what problem are we solving first? Something like "With CDK v2.0, we want to strictly adhere to semantic versioning. That means all experimental code will be in libraries with versions starting with 0.x, and this means the CDK 2.x libraries can only contain stable code".

I don't like that we're starting with a solution, instead of a problem. There are many ways to achieve the above goal, and details like "separate repository" are kind of orthogonal to that concern.

Reply:
I put that as the summary because the RFC is recommending that solution. I feel that mixing experiments into the stable repo is a slippery slope, and enforcing the separation is the best way to make sure the 2.x releases are all stable.

Reply:
I agree with @ericzbeard. The summary should succinctly describe the "result" of the RFC assuming some people will only read this to determine if they are actually interested to dive deeper.

Reply:
At the same time I agree with @skinny85  that it's super helpful to have a clear & crisp problem statement somewhere.